src/passes/libcore.rs

```diff
@@ -0,0 +1,47 @@
+use std::path::{Path, PathBuf};
+
+use crate::args::Args;
+use crate::copy_rs_files;
+use crate::error::Error;
+use crate::passes::{Pass, TestCase};
+
+pub enum LibCore {
+    // libcore from rustc 1.49.0
+    V149,
+}
+
+impl LibCore {
+    fn as_str(&self) -> &str {
+        match self {
+            LibCore::V149 => "1.49",
+        }
+    }
+}
+
+impl Pass for LibCore {
+    fn fetch(&self, args: &Args) -> Result<Vec<PathBuf>, Error> {
+        let rust_path = &args.rust_path;
+        let core_path = rust_path.join("library").join("core");
+
+        // TODO: How do we ensure the rustc submodule contains 1.49.0 libcore?
```

Comment:
`git checkout 1.49.0` with `Command::new`?

Reply:
Yeah, was thinking of just spawning it but I wanted to maybe leave it for later when we actually need multiple versions of libcore to test. It would lower the mental load for sure so I might do it now